firmware/tunerstudio/rusefi.input

```diff
@@ -3774,6 +3783,28 @@ cmd_set_engine_type_default					= "@@TS_IO_TEST_COMMAND_char@@@@ts_command_e_TS_
 		field = "SPI3sck mode",							spi3SckMode, {is_enabled_spi_3 == 1}@@if_ts_show_spi3_pins
 
 		field = "SPI4 enable",							is_enabled_spi_4
+		field = "SPI4 MOSI",							spi4mosiPin, {is_enabled_spi_4 == 1}@@if_ts_show_spi4_pins
+		field = "SPI4mosi mode",						spi4MosiMode, {is_enabled_spi_4 == 1}@@if_ts_show_spi4_pins
```

Comment:
let's NOT add those "*Mode* fields

we might actually remove existing "*Mode* fields one day